bug: login fails on Sony TV Android 11 Termux

### Bug description

I tried login in using otp and password both. both failed:


`Sending OTP to your mobile number
OTP sent to your mobile number
Enter OTP: 795671
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x85922c0]

goroutine 1 [running]:
log.(*Logger).output(0x0, 0x0, 0x2, 0xa02f91c)
	log/log.go:203 +0x3c
log.(*Logger).Println(...)
	log/log.go:276
github.com/jiotv-go/jiotv_go/v3/pkg/utils.GetDeviceID()
	github.com/jiotv-go/jiotv_go/v3/pkg/utils/utils.go:331 +0xb0
github.com/jiotv-go/jiotv_go/v3/pkg/utils.LoginVerifyOTP({0xa09b180, 0xd}, {0xa34e158, 0x6})
	github.com/jiotv-go/jiotv_go/v3/pkg/utils/utils.go:126 +0x8c
github.com/jiotv-go/jiotv_go/v3/cmd.LoginOTP()
	github.com/jiotv-go/jiotv_go/v3/cmd/login.go:69 +0x330
main.main.func5(0xa0a94c0)
	github.com/jiotv-go/jiotv_go/v3/main.go:168 +0x14
github.com/urfave/cli/v2.(*Command).Run(0xa0e6548, 0xa0a94c0, {0xa09cae0, 0x1, 0x1})
	github.com/urfave/cli/v2@v2.27.5/command.go:276 +0x7b4
github.com/urfave/cli/v2.(*Command).Run(0xa0e6488, 0xa0a9440, {0xa0aec00, 0x2, 0x2})
	github.com/urfave/cli/v2@v2.27.5/command.go:269 +0xaac
github.com/urfave/cli/v2.(*Command).Run(0xa0e6b48, 0xa0a9380, {0xa0b0030, 0x3, 0x3})
	github.com/urfave/cli/v2@v2.27.5/command.go:269 +0xaac
github.com/urfave/cli/v2.(*App).RunContext(0xa082c68, {0x8b30108, 0x8f27ce0}, {0xa0b0030, 0x3, 0x3})
	github.com/urfave/cli/v2@v2.27.5/app.go:333 +0x62c
github.com/urfave/cli/v2.(*App).Run(...)
	github.com/urfave/cli/v2@v2.27.5/app.go:307
main.main()
	github.com/jiotv-go/jiotv_go/v3/main.go:261 +0x1d60`

### Steps to reproduce

take a sony tv, install termux, use auto script to install, try login in with otp

### Relevant log output

```shell
Sending OTP to your mobile number
OTP sent to your mobile number
Enter OTP: 795671
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x85922c0]

goroutine 1 [running]:
log.(*Logger).output(0x0, 0x0, 0x2, 0xa02f91c)
	log/log.go:203 +0x3c
log.(*Logger).Println(...)
	log/log.go:276
github.com/jiotv-go/jiotv_go/v3/pkg/utils.GetDeviceID()
	github.com/jiotv-go/jiotv_go/v3/pkg/utils/utils.go:331 +0xb0
github.com/jiotv-go/jiotv_go/v3/pkg/utils.LoginVerifyOTP({0xa09b180, 0xd}, {0xa34e158, 0x6})
	github.com/jiotv-go/jiotv_go/v3/pkg/utils/utils.go:126 +0x8c
github.com/jiotv-go/jiotv_go/v3/cmd.LoginOTP()
	github.com/jiotv-go/jiotv_go/v3/cmd/login.go:69 +0x330
main.main.func5(0xa0a94c0)
	github.com/jiotv-go/jiotv_go/v3/main.go:168 +0x14
github.com/urfave/cli/v2.(*Command).Run(0xa0e6548, 0xa0a94c0, {0xa09cae0, 0x1, 0x1})
	github.com/urfave/cli/v2@v2.27.5/command.go:276 +0x7b4
github.com/urfave/cli/v2.(*Command).Run(0xa0e6488, 0xa0a9440, {0xa0aec00, 0x2, 0x2})
	github.com/urfave/cli/v2@v2.27.5/command.go:269 +0xaac
github.com/urfave/cli/v2.(*Command).Run(0xa0e6b48, 0xa0a9380, {0xa0b0030, 0x3, 0x3})
	github.com/urfave/cli/v2@v2.27.5/command.go:269 +0xaac
github.com/urfave/cli/v2.(*App).RunContext(0xa082c68, {0x8b30108, 0x8f27ce0}, {0xa0b0030, 0x3, 0x3})
	github.com/urfave/cli/v2@v2.27.5/app.go:333 +0x62c
github.com/urfave/cli/v2.(*App).Run(...)
	github.com/urfave/cli/v2@v2.27.5/app.go:307
main.main()
	github.com/jiotv-go/jiotv_go/v3/main.go:261 +0x1d60
```


### Screenshots or videos

_No response_

### Solution

_No response_

### Additional context

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

--- a/cmd/login.go
+++ b/cmd/login.go
@@ -42,6 +42,7 @@ func Logout() error {
 // verifies the entered OTP by the user and logs in the user.
 // Returns any error encountered.
 func LoginOTP() error {
+	utils.Log = utils.GetLogger()
 	err := store.Init()
 	if err != nil {
 		return err
@@ -87,6 +88,7 @@ func LoginOTP() error {
 // and logs in the user if successful.
 // Returns any error encountered.
 func LoginPassword() error {
+	utils.Log = utils.GetLogger()
 	err := store.Init()
 	if err != nil {
 		return err

